app/http.php

```diff
@@ -500,10 +498,81 @@ function removeAllRuntimes(Table $activeRuntimes, Orchestration $orchestration):
                 network: \strval(Http::getEnv('OPR_EXECUTOR_NETWORK', 'executor_runtimes')),
                 workdir: $workdir
             );
+            var_dump("container has started");
 
             if (empty($containerId)) {
                 throw new Exception('Failed to create runtime', 500);
             }
+        } catch (Throwable $th) {
+            $error = $th->getMessage() . $output;
+
+            // Extract as much logs as we can
+            // try {
+            //     $logs = '';
+            //     $status = $orchestration->execute(
+            //         name: $runtimeName,
+            //         command: ['sh', '-c', 'cat /var/tmp/logs.txt'],
+            //         output: $logs,
+            //         timeout: 15
+            //     );
+
+            //     if (!empty($logs)) {
+            //         $error = $th->getMessage() . $logs;
+            //     }
+            // } catch (Throwable $err) {
+            //     // Ignore, use fallback error message
+            // }
+
+            if ($remove) {
+                \sleep(2); // Allow time to read logs
+            }
+
+            $localDevice->deletePath($tmpFolder);
+
+            // Silently try to kill container
+            try {
+                $orchestration->remove($runtimeName, true);
+            } catch (Throwable $th) {
+            }
+
+            $activeRuntimes->del($runtimeName);
+
+            throw new Exception($error, 500);
+        }
+
+        $response
+            ->setStatusCode(Response::STATUS_CODE_CREATED)
+            ->json($container);
+    });
+
+Http::post('/v1/runtimes/:runtimeId/builds')
```

Comment:
```suggestion
Http::post('/v1/runtimes/:runtimeId/commands')
```

It would be more classic to call it executions as this is the underline Docker command (`docker exec`), but commands can also work. I would avoid builds as its the Appwrite specific use case.